packages/grafana-ui/src/components/Divider/Divider.tsx

```diff
@@ -1,17 +1,18 @@
 import { css } from '@emotion/css';
 import React from 'react';
 
-import { GrafanaTheme2 } from '@grafana/data';
+import { GrafanaTheme2, ThemeSpacingTokens } from '@grafana/data';
 
 import { useTheme2 } from '../../themes';
 
 interface DividerProps {
   direction?: 'vertical' | 'horizontal';
+  spacing?: ThemeSpacingTokens;
 }
 
-export const Divider = ({ direction = 'horizontal' }: DividerProps) => {
+export const Divider = ({ direction = 'horizontal', spacing = 2 }: DividerProps) => {
   const theme = useTheme2();
-  const styles = getStyles(theme);
+  const styles = getStyles(theme, spacing);
```

Comment:
let's memoize this properly with `useStyles2` + `useCallback` (see https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Badge/Badge.tsx#L22 for an example)